gobblin-yarn/src/test/java/org/apache/gobblin/yarn/YarnServiceTest.java

```diff
@@ -197,8 +201,12 @@ public void tearDown() throws IOException, TimeoutException, YarnException {
   public void testScaleUp() {
     this.yarnService.requestTargetNumberOfContainers(10, Collections.EMPTY_SET);
 
+    Assert.assertFalse(this.yarnService.getMatchingRequestsList(64, 1).isEmpty());
     Assert.assertEquals(this.yarnService.getNumRequestedContainers(), 10);
     Assert.assertTrue(this.yarnService.waitForContainerCount(10, 60000));
+
+    // container request list that had entries earlier should now be empty
+    Assert.assertEquals(this.yarnService.getMatchingRequestsList(64, 1).size(), 0);
```

Comment:
Assert.assertTrue(this.yarnService.getMatchingRequestsList(64, 1).isEmpty())? to be consistent with line 204?

Reply:
The comparison with size is done so that the mismatching size is printed in the error case which may give an idea about the error.